docs/en/connector-v2/sink/OssFile.md

```diff
@@ -0,0 +1,203 @@
+# OssFile
+
+> Oss file sink connector
+
+## Description
+
+Output data to oss file system.
+
+> Tips: We made some trade-offs in order to support more file types, so we used the HDFS protocol for internal access to OSS and this connector need some hadoop dependencies.
+> It's only support hadoop version **2.9.X+**.
+
+## Options
+
+| name                             | type   | required | default value                                             |
+|----------------------------------| ------ |---------|-----------------------------------------------------------|
+| path                             | string | yes     | -------                                                   |
+| bucket                           | string | yes     | -------                                                   |
+| access_key                       | string | yes     | -------                                                   |
+| access_secret                    | string | yes     | -------                                                   |
+| endpoint                         | string | yes     | -------                                                   |
+| file_name_expression             | string | no      | "${transactionId}"                                        |
+| file_format                      | string | no      | "text"                                                    |
+| filename_time_format             | string | no      | "yyyy.MM.dd"                                              |
+| field_delimiter                  | string | no      | '\001'                                                    |
+| row_delimiter                    | string | no      | "\n"                                                      |
+| partition_by                     | array  | no      | -                                                         |
```

Comment:
`-` is the default partition key? Or `-` is seem as `-------` only mean there is no default value?